src/routes/event-team.tsx

```diff
@@ -195,11 +184,36 @@ const EventTeam = ({ eventKey, teamNum }: Props) => {
       back={`/events/${eventKey}`}
       class={eventTeamStyle}
     >
-      {nextMatch && (
+      {upcomingMatches && upcomingMatches.length > 0 ? (
         <>
-          <h2 class={sectionStyle}>Next Match</h2>
-          <MatchDetailsCard match={nextMatch} eventKey={eventKey} link />
+          <Heading level={2}>Upcoming Matches</Heading>
```

Comment:
This is a little nitpicky, but the space between the "upcoming matches" heading and the actual match cards is half the size of the space between the other elements on the page.

Before this PR:

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/80c360ae-557d-49e4-ba09-ad811e91d899)

After this PR:

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/89841bda-58dd-49b2-a2cf-c32e706fd24a)

The new heading looks nice in terms of the underline and text styling, but it feels a little bit cramped without the even space around it.